sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/Analyzer.scala

```diff
@@ -209,13 +209,8 @@ class Analyzer(override val catalogManager: CatalogManager) extends RuleExecutor
     if (plan.analyzed) return plan
     AnalysisHelper.markInAnalyzer {
       val analyzed = executeAndTrack(plan, tracker)
-      try {
-        checkAnalysis(analyzed)
-        analyzed
-      } catch {
-        case e: AnalysisException =>
-          throw new ExtendedAnalysisException(e, analyzed)
-      }
```

Comment:
The removed code seems to be used very long time. When we remove `try ... catch ...` here, I'm wondering if there is any chance of side effect in the following which currently throws `ExtendedAnalysisException` instead of `AnalysisException`?

https://github.com/apache/spark/blob/742a892d58e4e5fc960a58dc4c7e12e9c41073bd/sql/core/src/main/scala/org/apache/spark/sql/execution/QueryExecution.scala#L87